apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx

```diff
@@ -21,22 +21,20 @@ import {useTokenInfo} from '../yoroi-wallets/hooks'
 
 export const ActionsBanner = ({disabled = false}: {disabled: boolean}) => {
   const {styles, colors} = useStyles()
-
-  const ACTION_PROPS = {
-    size: 24,
-    color: String(colors.actionColor),
-  }
-
   const strings = useStrings()
-  const {isSingleAddress} = useMultipleAddresses()
   const navigateTo = useNavigateTo()
-  const wallet = useSelectedWallet()
+
+  const {isSingle, addressDerivation} = useAddressDerivationManager()
+  const {next: nextReceiveAddress} = useReceiveAddressesStatus(addressDerivation)
```

Comment:
on ActionsBanner, there is no receive address selected, only the `next` that depends on wallet state, the context is only for the details, since we avoid params.